spec/koans.js

```diff
@@ -86,40 +86,48 @@ describe('`string.includes()` finds string within another string. ', () => {
   describe('find a single character', function() {
     it('in a three char string', function() {
       /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+      const searchString =  "x"   //.includes("xyz");
+      expect('xyz'.includes(searchString)).toBe(true);
     });
     it('reports false if character was not found', function() {
       /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+      const expected = false
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
   describe('find a string', function() {
     it('that matches exactly', function() {
+      const findSome = parameter => 'xyz'.includes();
       /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+        // function findSome(par) {
+        // return "xyz".includes(par)
+        //   }
+      
+      const findSome = sentence.includes('z')
+      expect(findSome('xyz')).toBe(true);
     });
   });
 
   describe('search for an empty string, is always true', function() {
     it('in an empty string', function() {
-      /* .... */
-      //expect(''.includes(x)).toBe(true);
+      /* .... */ const x = ''
+      expect(''.includes(x)).toBe(true);
     });
     it('in `abc`', function() {
-      /* .... */
-      //expect('abc'.includes(x)).toBe(true);
+      /* .... */const x ='abc'
```

Comment:
here you can write again
```js
const x = ''
```
the point is, in javascript if you search for an ampty string inside a string with characters, it will always return true.